tools/update_gene_symbols.cwl

```diff
@@ -0,0 +1,38 @@
+cwlVersion: v1.0
+class: CommandLineTool
+id: update-gene-symbols
+requirements:
+  - class: ShellCommandRequirement
+  - class: InlineJavascriptRequirement
+  - class: ResourceRequirement
+    ramMin: 4000
+    coresMin: 2
+  - class: DockerRequirement
+    dockerPull: 'pgc-images.sbgenomics.com/d3b-bixu/hotspots:0.1.0'
```

Comment:
I think this will run just fine in any base python3 docker.

Reply:
Um, hmmm, I see, so you're saying just look for some public python3 docker? The only python-specific docker we have built other than that seems to be for v2.7 - I don't want to use that

Reply:
Yeah you can use a public python docker.